config/routes.rb

```diff
@@ -45,6 +45,8 @@
   end
 
   namespace :admin do
-    resources :slack_presence_checks, only: [:index]
+    resources :slack_presence_checks do
+      get '/page/:page', action: :index, on: :collection
+    end
```

Comment:
If we can change the links to use query params we can keep the same routing

Reply:
routes are reverted and passing tests